Application/EdFi.Ods.SandboxAdmin/Services/PasswordService.cs

```diff
@@ -48,12 +48,10 @@ public async Task<PasswordResetResult> ValidateRequest(string userEmail, string
                 return PasswordResetResult.BadUsername;
             }
 
-            if (!await _identityProvider.VerifyUserEmailConfirmed(userEmail))
+            if (!await _identityProvider.VerifyUserEmailConfirmed(userEmail)
+                 && !await _identityProvider.VerifyUserPassword(userEmail, secret))
```

Comment:
Code smell recommendation: unify the if statements. I agreed with that and applied this fix manually. This is not covered by `dotnet format`.